runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4135,6 +4135,12 @@ done:;
 			goto done;
 		}
 		_vm->anonClassLoader->flags |= J9CLASSLOADER_ANON_CLASS_LOADER;
+		/* ShareAnonymousClasses is enabled by default so we set the class loader flag */
+		_vm->anonClassLoader->flags |= J9CLASSLOADER_SHARED_CLASSES_ENABLED;
```

Comment:
No need to set J9CLASSLOADER_SHARED_CLASSES_ENABLED if _vm->sharedClassConfig is NULL.